CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org)
 
+## 2.2.0 - 2019-09-26
```

Comment:
This line should say `## MASTER` until a release is tagged because this is going to be on `2.1.1-dev`.

Reply:
Well hey then we get to give Scrutinizer another go at it :)